x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.tsx

```diff
@@ -104,453 +104,463 @@ const StyledEuiCodeBlock = euiStyled(EuiCodeBlock).attrs({
-    }, [agentIds, getTestId, itemIdToExpandedRowMap, onClickCallback]);
-
-    // table pagination
-    const tablePagination = useMemo(() => {
-      return {
-        // this controls the table UI page
-        // to match 0-based table paging
-        pageIndex: (queryParams.page || 1) - 1,
-        pageSize: queryParams.pageSize || 10,
-        totalItemCount,
-        pageSizeOptions: MANAGEMENT_PAGE_SIZE_OPTIONS as number[],
-      };
-    }, [queryParams, totalItemCount]);
-
-    // handle onChange
-    const handleTableOnChange = useCallback(
-      ({ page: _page }: CriteriaWithPagination<ActionDetails>) => {
-        // table paging is 0 based
-        const { index, size } = _page;
-        setQueryParams((prevState) => ({
-          ...prevState,
-          // adjust the page to conform to
-          // 1-based API page
-          page: index + 1,
-          pageSize: size,
-        }));
-        reFetchEndpointActionList();
       },
-      [reFetchEndpointActionList, setQueryParams]
-    );
-
-    // compute record ranges
-    const pagedResultsCount = useMemo(() => {
-      const page = queryParams.page ?? 1;
-      const perPage = queryParams?.pageSize ?? 10;
-
-      const totalPages = Math.ceil(totalItemCount / perPage);
-      const fromCount = perPage * page - perPage + 1;
-      const toCount =
-        page === totalPages || totalPages === 1 ? totalItemCount : fromCount + perPage - 1;
-      return { fromCount, toCount };
-    }, [queryParams.page, queryParams.pageSize, totalItemCount]);
-
-    // create range label to display
-    const recordRangeLabel = useMemo(
-      () => (
-        <EuiText color="subdued" size="xs" data-test-subj={getTestId('endpointListTableTotal')}>
-          <FormattedMessage
-            id="xpack.securitySolution.responseActionsList.list.recordRange"
-            defaultMessage="Showing {range} of {total} {recordsLabel}"
-            values={{
-              range: (
-                <strong>
-                  <EuiI18nNumber value={pagedResultsCount.fromCount} />
-                  {'-'}
-                  <EuiI18nNumber value={pagedResultsCount.toCount} />
-                </strong>
-              ),
-              total: <EuiI18nNumber value={totalItemCount} />,
-              recordsLabel: <strong>{UX_MESSAGES.recordsLabel(totalItemCount)}</strong>,
-            }}
-          />
-        </EuiText>
-      ),
-      [getTestId, pagedResultsCount.fromCount, pagedResultsCount.toCount, totalItemCount]
-    );
-
-    return (
-      <>
-        <ActionsLogFilters
-          dateRangePickerState={dateRangePickerState}
-          isDataLoading={isFetching}
-          onClick={reFetchEndpointActionList}
-          onChangeCommandsFilter={onChangeCommandsFilter}
-          onRefresh={onRefresh}
-          onRefreshChange={onRefreshChange}
-          onTimeChange={onTimeChange}
+      {
+        field: '',
+        align: RIGHT_ALIGNMENT as HorizontalAlignment,
+        width: '40px',
```

Comment:
just FYI - I did not add up all of the percentages, but I assume they don't go over 100%, right?

Reply:
Yeah I'm keeping the total percentages to 99 and then the 40px for the expander column